manifests/g/GNU/Icecat/91.4.1/GNU.Icecat.yaml

```
@@ -0,0 +1,15 @@
+PackageIdentifier: GNU.Icecat
+PackageVersion: 91.4.1
+PackageName: icecat
+Publisher: GNU.org
```

This must match what is shown in control panel
```suggestion
Publisher: Mozilla
```